core/domain/customization_args_util_test.py

```diff
@@ -390,48 +399,41 @@ def test_frontend_customization_args_defs_coverage(self):
                     line
                 ))
             if ca_frontend_interface_match:
-                actual_interaction_ca_frontend_interfaces.add(
+                interaction_ids_with_ca_frontend_interfaces.add(
                     ca_frontend_interface_match.group(2))
 
-        expected_interaction_ca_interfaces = (
-            set(interaction_registry.Registry.get_all_interaction_ids()))
-
+        self.assertTrue(len(interaction_ids_with_ca_backend_interfaces) > 0)
```

Comment:
Next PR -- Here and below: assertGreater

Reply:
Done